diff --git a/dist/main/lib/webauthn.dom.d.ts b/dist/main/lib/webauthn.dom.d.ts
index 2b004e46b6a469572989fed4e10c644fff42fa3d..80e166a7c0e7fa1d15ca334de0c35128f4022e7d 100644
--- a/dist/main/lib/webauthn.dom.d.ts
+++ b/dist/main/lib/webauthn.dom.d.ts
@@ -498,7 +498,7 @@ export type PublicKeyCredentialJSON = RegistrationResponseJSON | AuthenticationR
  * @see {@link https://w3c.github.io/webauthn/#publickeycredential W3C WebAuthn Spec - PublicKeyCredential}
  * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential MDN - PublicKeyCredential}
  */
-export interface PublicKeyCredentialFuture<T extends PublicKeyCredentialJSON = PublicKeyCredentialJSON> extends PublicKeyCredential {
+export interface PublicKeyCredentialFuture<T extends PublicKeyCredentialJSON = PublicKeyCredentialJSON> extends Omit<PublicKeyCredential, 'toJSON'> {
     /**
      * The type of the credential (always "public-key").
      * @see {@link https://w3c.github.io/webauthn/#dom-credential-type W3C - type}
diff --git a/dist/module/lib/webauthn.dom.d.ts b/dist/module/lib/webauthn.dom.d.ts
index 2b004e46b6a469572989fed4e10c644fff42fa3d..80e166a7c0e7fa1d15ca334de0c35128f4022e7d 100644
--- a/dist/module/lib/webauthn.dom.d.ts
+++ b/dist/module/lib/webauthn.dom.d.ts
@@ -498,7 +498,7 @@ export type PublicKeyCredentialJSON = RegistrationResponseJSON | AuthenticationR
  * @see {@link https://w3c.github.io/webauthn/#publickeycredential W3C WebAuthn Spec - PublicKeyCredential}
  * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential MDN - PublicKeyCredential}
  */
-export interface PublicKeyCredentialFuture<T extends PublicKeyCredentialJSON = PublicKeyCredentialJSON> extends PublicKeyCredential {
+export interface PublicKeyCredentialFuture<T extends PublicKeyCredentialJSON = PublicKeyCredentialJSON> extends Omit<PublicKeyCredential, 'toJSON'> {
     /**
      * The type of the credential (always "public-key").
      * @see {@link https://w3c.github.io/webauthn/#dom-credential-type W3C - type}
